carrot-test-config/carrot_commands.txt

```diff
@@ -1,52 +0,0 @@
-# Create a software entry for this repository
-carrot_cli software create --name example-test --description "An example CARROT test software" \
```

Comment:
Why get rid of this file? I found the step-by-step commands for setting up the carrot run to be very useful and exactly what i wanted out of this repo. Or am I missing that there is another one elsewehre that is better?

Reply:
If you are changing this you have to update the readme to point me to the right place since it still refers to this file.

Reply:
Sorry, the new file is there now in the new commit